lib/taskana-core/src/main/java/pro/taskana/common/internal/TaskanaEngineImpl.java

```diff
@@ -209,7 +208,7 @@ public boolean isUserInRole(TaskanaRole... roles) {
   public void checkRoleMembership(TaskanaRole... roles) throws NotAuthorizedException {
     if (!isUserInRole(roles)) {
       if (LOGGER.isDebugEnabled()) {
-        String accessIds = LoggerUtils.listToString(CurrentUserContext.getAccessIds());
+        String accessIds = CurrentUserContext.getAccessIds().toString();
```

Comment:
just use `CurrentUser.getAccessIds()` in the debug message. That way you don't have to use a `toString` call :)

Reply:
done :)